src/data-type.js

```diff
@@ -1305,10 +1305,30 @@ const TYPE = module.exports.TYPE = {
       if (value instanceof Date) {
         return value;
       }
-      value = Date.parse(value);
-      if (isNaN(value)) {
+      var dateValue = Date.parse(value);
+      if (!isNaN(dateValue)) {
```

Comment:
Mmmm it seems this validation will keep these inputs:

* "10:"
* "10:54:"

¿Is there a problem with it?